frontend/src/components/Footer/Footer.jsx

```diff
@@ -26,7 +26,7 @@ const footerBottomLinks = [
     link: '/donnees-personnelles-et-gestion-des-cookies',
   },
   { key: 'footer-bottom-link-3', title: 'Mentions légales', link: '/mentions-legales' },
```

Comment:
shouldn't we have a alt for all of them ?

Reply:
It's only necessary if the `title` is not explicit.

I added it for `En savoir plus` because in one case it's the FAQ, in another case (landing page) it's parcours etudiant PDF.

I'll check if there is a console error for the ones without alt, if no I keep it like this.